src/lib/Microsoft.Fx.Portability.Reports.DGML/ReferenceGraph.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Fx.Portability.ObjectModel;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Fx.Portability.Reports.DGML
+{
+    class ReferenceGraph
+    {
+        public Dictionary<ReferenceNode, ReferenceNode> Nodes { get; set; }
```

Comment:
Is this Nodes member set else where? If not, we can make this:
`public Dictionary<ReferenceNode, ReferenceNode> Nodes { get; }`